### PR TITLE
Disable spline potential

### PR DESCRIPTION
**Pull request by danielhollas**
ABIN has an implementation of a numerical 1D potential on top of user-defined grid of points (`pot="_splined_grid_"` ). We use cubic splines to interpolate between the points.

Unfortunately, the cubic spline routines (originally taken from QDYN) are from Numerical recipes and we don't have license for them. Until we implement our own cubic spline, this commit remove the NR routines and disables the `_spline_` potential and associated tests.

### CODE REVIEW

File: src/force_spline.F90 [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ! 1D user-defined numerical potential with cubic splines
 ! Original code taken from QDYN
 ! https://github.com/PHOTOX/qdyn
```
Comment (JanosJiri) [inline src/force_spline.F90:3]: Just a comment, the cubic splines are not implemented in QDyn anymore. It's probably still in some of the old commits.

**Review by JanosJiri — APPROVED** [review]
Looks good, I was just surprised you can't use splines from numerical recipes.

File: src/force_spline.F90 (hunk 1 above)
Comment (danielhollas) [reply src/force_spline.F90:3]: Good to know. What do you use instead?

Comment (JanosJiri) [reply src/force_spline.F90:3]: I use Python scripts to prepare the potential.

**Pull request merged**